### DISCUSSION TITLE
TransformedTargetForecaster Pipeline update

### DISCUSSION THREAD

**Discussion by yadeesha88** [alan-turing-institute/sktime#1467] [Q&A]
Hi all,
I created a TransformedTargetForecaster pipeline as below.

`forecaster_pipeline = TransformedTargetForecaster(
    [
        ("deseasonal", deseasonal_transformer),
        ("detrend", detrend_transformer),
        ("forecast", forecaster),
    ]
)`

As I understand, when I tried to use `forecaster_pipeline.update(y_update)` it only updates the detrend and deseasonal models and doesn't transform `y_update` (unlike `forecaster_pipeline.fit()`). Therefore, the input to the forecast model is now `y_update` untransformed unlike during `fit()` where the input to the model are already transformed. This causes erroneous results. Appreciate if someone can help me to avoid this.

Thank you!

**Comment by fkiraly** [comment]
Hm, that doesn't sound right - @aiwalter, can you look into this?
If it's a bug, you could fix it as part of #1376 to ensure it also works correctly in the multivariate case?

**Reply by aiwalter** [reply]
sure, let me look into that!

**Comment by mloning** [comment]
Thanks @yadeesha88 - there's already an issue on this here: https://github.com/alan-turing-institute/sktime/issues/1381

**Reply by yadeesha88** [reply]
Thank you very much @fkiraly @aiwalter @mloning .
